_rules/first-focusable-element-is-link-to-non-repeated-content-8a213c.md

```diff
@@ -0,0 +1,658 @@
+---
+id: 8a213c
+name: First focusable element is link to non-repeated content
+rule_type: atomic
+description: |
+  This rule checks that the first focusable element is a link to non-repeated content in the page
+accessibility_requirements:
+  wcag-technique:G1: # Adding a link at the top of each page that goes directly to the main content area
+    forConformance: false
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility Tree
+  - CSS styling
+  - DOM Tree
+  - Language
+acknowledgments:
+  authors:
+    - Christina Adams
+    - Jean-Yves Moyen
+  assets:
+    - _The Three Kingdoms_ by Luo Guanzhong, translation by Yu Sumei (Tuttle publishing, 2014, ISBN 9780804843935)
+---
+
+## Applicability
+
+This rule applies to any [HTML web page][].
+
+## Expectation 1
+
+Each test target has a non-empty [sequential focus navigation order][sequential focus navigation].
+
+## Expectation 2
+
+For each test target, all the following are true for the first element in its [sequential focus navigation order][sequential focus navigation]:
+
+- the element is [keyboard actionable][]; and
+- the element is [included in the accessibility tree][]; and
+- the element is a [semantic link][]; and
+- when the element is [activated][], focus moves [just before][] a node of [non-repeated content][]; and
+- the element has an [accessible name][] that communicates that it skips to non-repeated content.
```

Comment:
Do we want to check that the accessible name communicates that it "skips to non-repeated content"? I never seen a page where the skip link communicates that :-)
They communicate that it skips to main or primary or something like that. Going from that to non-repeated content is an indirection...
```suggestion
- the element has an [accessible name][] that communicates that it skips to the primary content.
```

Reply:
🤔 
I was a bit afraid that keeping the name to describe "skip to main" ends up being too disconnected from the rest of the expectation since it would be the only place in the rule where "primary content" or similar is mentioned. But I guess you're right and we should stick to it.